docs/basic/index.rst

```diff
@@ -91,17 +91,21 @@ You will need to configure your system to register with and trust packages from
 the CrateDB package repository::
 
     # Install prerequisites.
-    apt-get install sudo
-    sudo apt-get install curl gnupg software-properties-common apt-transport-https apt-utils
+    apt update
+    apt install --yes sudo
```

Comment:
Do we need this step?

Reply:
No, it is actually an optional step, when we use `sudo` throughout the tutorial. Let's just remove it?

Reply:
Without sudo you would not be able to update `apt update`

Reply:
I've just submitted 8fda19612c, implementing my suggestion.